include/RAJA/util/Operators.hpp

```diff
@@ -336,16 +335,16 @@ static_assert(check<unsigned long long>(),
 template <typename Ret, typename Arg1 = Ret, typename Arg2 = Arg1>
 struct plus : public detail::binary_function<Arg1, Arg2, Ret>,
               detail::associative_tag {
-  RAJA_HOST_DEVICE Ret operator()(const Arg1& lhs, const Arg2& rhs)
+  RAJA_HOST_DEVICE Ret operator()(const Arg1& lhs, const Arg2& rhs) const
```

Comment:
well, I feel dumb about making these non-const originally :\

Reply:
The surprise on there was that they ever worked without it... with lambda capture all over the place, everything becomes const *all the time*.  Not sure what tickled it, but something did. 🤷‍♂️